app/services/ember-cli.js

```diff
@@ -503,10 +510,16 @@ export default Ember.Service.extend({
  */
 function babelOpts(moduleName) {
   return {
-    modules: 'amdStrict',
+    presets: ['es2017'],
     moduleIds: true,
     moduleId: moduleName,
-    plugins: [ hbsPlugin ]
+    plugins: [
+      ['transform-es2015-modules-amd', {
+        loose: true,
+        noInterop: false
```

Comment:
If you set this to `true`, I do not think that you will need the `__esModule = true` flag above (because Babel will not check if it has to interop).

Reply:
Needed for compat with addons compiled with babel@5 using modules: 'amdStrict'